squash_wip_test.go

```diff
@@ -141,7 +141,7 @@ func TestCommitsOnCurrentBranch(t *testing.T) {
 	commits := commitsOnCurrentBranch(configuration)
 
 	equals(t, []string{
-		configuration.WipCommitMessage,
+		configuration.WipCommitMessage + " lastFile:file3.txt",
```

Comment:
Hm, I think the test here should fail because of the different commit message. Why does it not fail?

Reply:
This returned just the subject. If you have a subject with one linebreak this gets replaced with an whitespace by git. Now the lastFile is not part of the subject anymore, so this can be removed.